GliaWidgets/Sources/Extensions/Data+Extension.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+extension Data {
+    var html2AttributedString: NSAttributedString? {
+        do {
+            return try NSAttributedString(
+                data: self,
+                options: [.documentType: NSAttributedString.DocumentType.html, .characterEncoding: String.Encoding.utf8.rawValue],
+                documentAttributes: nil
+            )
+        } catch {
+            print("error:", error)
```

Comment:
Would be good to have more descriptive prefix, indicating that html-string decoding failed

Reply:
And if we don't log it to Kibana, would be better to prevent logging to console in prod env. So `debugPrint` would be preferable

Reply:
fixed [9c3c9a7](https://github.com/salemove/ios-sdk-widgets/pull/677/commits/9c3c9a71f8cead51b40cadc03f7ecf99a111eb4b)